datarepo-clienttests/src/main/resources/configs/profiling/BuildSnapshotWithFiles1000_50.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "BuildSnapshotWithFiles050",
```

Comment:
should this have the same name as the file below?
datarepo-clienttests/src/main/resources/configs/profiling/BuildSnapshotWithFiles1000_100.json